app/navigation/JourneyTypeNavigator.scala

```diff
@@ -55,6 +56,13 @@ class JourneyTypeNavigator @Inject()() extends BaseNavigator {
     case CheckYourAnswersJourneyTypePage => (userAnswers: UserAnswers) =>
       routes.DraftMovementController.onPageLoad(userAnswers.ern, userAnswers.arc)
 
+    case JourneyTypeReviewPage => (userAnswers: UserAnswers) =>
```

Comment:
Not related to your change, but you might want to add `//noinspection ScalaStyle - Cyclomatic Complexity` here